recipes/tidy-html5/all/conanfile.py

```diff
@@ -1,17 +1,22 @@
-from conans import ConanFile, CMake, tools
 import os
 
-required_conan_version = ">=1.33.0"
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.microsoft import is_msvc
+
+required_conan_version = ">=1.53.0"
 
 
 class TidyHtml5Conan(ConanFile):
     name = "tidy-html5"
+    description = "The granddaddy of HTML tools, with support for modern standards"
     license = "W3C"
```

Comment:
Just checked, and the library does not seem to actually be licensed under these terms:

https://spdx.org/licenses/W3C.html differs substantially from https://github.com/htacg/tidy-html5/blob/next/README/LICENSE.md

Reply:
Indeed. Turns out their license actually exists as a separate SPDX liecense ID: [`HTMLTIDY`](https://spdx.org/licenses/HTMLTIDY.html)